_templates/exchanges.html

```diff
@@ -422,3 +422,6 @@ <h3 id="south-africa" class="no_toc">South Africa</h3>
 <p class="introlink exchanges-introlink">Visit
   <a href="https://coinatmradar.com/">Coin ATM Radar</a> to find local Bitcoin ATMs.
 </p>
+<p class="introlink exchanges-introlink">Visit
+  <a href="https://cryptoradar.co">Cryptoradar</a> to compare above exchanges based on prices, fees and features.
```

Comment:
Let's combine with with the first link to Buy Bitcoin Worldwide and write it like so:

![image](https://user-images.githubusercontent.com/1130872/42529810-ab9810a6-843c-11e8-944e-f02c41096278.png)